packages/babel-preset-kyt-core/lib/index.js

```diff
@@ -1,15 +1,26 @@
 var babelPresetLatest = require('babel-preset-latest');
 var babelTransformRuntime = require('babel-plugin-transform-runtime');
 var babelTransformModules = require('babel-plugin-transform-es2015-modules-commonjs');
+var babelSyntaxDynamicImport = require('babel-plugin-syntax-dynamic-import');
+var babelEslint = require('babel-eslint');
```

Comment:
I'm trying to add babel-eslint here in packages/babel-preset-kyt-core, but kyt itself is linted with packages/eslint-config-kyt – so I'm guessing I have to add babel-eslint and babel-plugin-syntax-dynamic-import to eslint-config-kyt as well.